pkg/kev/kev_reconcile_test.go

```diff
@@ -261,9 +268,21 @@ var _ = Describe("Reconcile", func() {
 				})
 
 				It("should configure the added service labels from deploy config", func() {
```

Comment:
The title of this `It` block should also change now?

Reply:
can we do this later? I'll find all cases and updated as I mentioned earlier.

Reply:
OK, I think It's a tiny change and so probably best to address on the spot? As we keep adding those changes it'll be hard to unpick that later I think.